packages/base-styles/_colors.scss

```diff
@@ -25,6 +25,15 @@ $light-gray-200: #f3f4f5;
 $light-gray-100: #f8f9f9;
 $white: #fff;
 
+// G2 colors.
+$dark-gray-primary: #1e1e1e;
+$medium-gray-text: #757575;
+$light-gray-secondary: #ccc;
+$light-gray-tertiary: #e7e8e9;
```

Comment:
Do you think we should replace all shared of grays (all gray variables) with these four variables?

Reply:
Yes, but I think it's also an opportunity to reduce the amount of grays we have. The wide range has not proven as useful in practice, given that they are intended to colorize the UI, vs. for example colorize a theme.